lib/xclarity_client/xclarity_resource.rb

```diff
@@ -4,6 +4,7 @@ def build_resource(attributes)
       attributes.each do |key, value|
         begin
           value = value.gsub("\u0000", '') if value.is_a?(String)
+          key = key.gsub("-","_")
```

Comment:
There are some properties coming from LXCA REST api which contains "-", so is necessary replace it for something to keep properties syntactically correct in model. Then I suggested @maysamacedo to implement this,  replacing "-" for "_".